usr/bin/cylc

```diff
@@ -191,7 +191,7 @@ elif [[ -d "${CYLC_HOME}/conda-meta" && \
         -f "${CYLC_HOME%/*/*}/etc/profile.d/conda.sh" ]]; then
     # A normal Conda environment
     . "${CYLC_HOME%/*/*}/etc/profile.d/conda.sh"
-    conda activate "${CYLC_HOME##*/}" || exit 1
+    # conda activate "${CYLC_HOME##*/}" || exit 1
```

Comment:
Huh, simple as that, is it?!

Reply:
As far as I can tell - I was confused by it, so I asked @oliver-sanders  to look over my shoulder...

Reply:
```suggestion
```

Reply:
```suggestion
```

Reply:
Yep, truely that simple, just call the executable using it's absolute path, Python automatically inserts the `lib/python<v>/site-packages/` (relative the the Python executable location) into `sys.path`.